tasks/test_results_processor.py

```diff
@@ -117,7 +117,7 @@ def compute_name(
                     if candidate.test_file_path == filename or (
                         network is not None and candidate.test_file_path in network
                     ):
-                        return f"{candidate.test_file_path}::{candidate.actual_class_name}::{raw_name}"
+                        return f"{candidate.test_file_path}::{f"{candidate.actual_class_name}::" if candidate.actual_class_name else ""}{raw_name}"
```

Comment:
nesting these format strings is a bit hard to read, can you move that into a separate variable?